_emails/kalamazooMi.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Kalamazoo, MI
+permalink: "/kalamazooMi"
```

Comment:
Let's just leave this at /kalamazoo for now?

Reply:
what should be the convention for more common city names if they appear?